scopes/component/checkout/checkout-cmd.ts

```diff
@@ -43,18 +43,18 @@ export class CheckoutCmd implements Command {
       'interactive-merge',
       'when a component is modified and the merge process found conflicts, display options to resolve them',
     ],
-    ['o', 'ours', 'in case of a conflict, override the used version with the current modification'],
-    ['t', 'theirs', 'in case of a conflict, override the current modification with the specified version'],
-    ['m', 'manual', 'in case of a conflict, leave the files with a conflict state to resolve them manually later'],
+    ['o', 'ours', 'in case of merge conflict, override incoming version with the current modification'],
+    ['t', 'theirs', 'in case of merge conflict, override the current modifications with incoming version'],
+    ['m', 'manual', 'in case of merge conflict, leave the files in conflict state to resolve them manually later'],
     ['r', 'reset', 'revert changes that were not snapped/tagged'],
     ['a', 'all', 'all components'],
     [
       'e',
       'workspace-only',
-      'when on a lane, avoid introducing new components from the remote lane that do not exist locally',
+      "when on a lane, don't import components from the remote lane that are not already in the workspace", // @david what's the connection between the `bit checkout` command and lanes?
```

Comment:
You can run `bit checkout head` when you're on a lane.
By default, we import the remote lane and checkout all components from the lane. 
If you use this flag, we ignore the components that are not in the workspace.

Reply:
So if i'm on a lane and run `bit checkout head` bit will by default start bringing in other components that aren't in my workspace?
IMO this is not good dev ex, i wouldnt expect `bit checkout` to do some lane-specific action. @itaymendel @GiladShoham wdut?

What about a `bit lane checkout` command to do this? I.e. if i run that without specifying a lane, then it checks out the current lane to its head, and then i can add the `--workspace-only` flag there to make sure it doesnt import components not already in the lane.
I would also add another flag `--lane-only` which will remove components from my workspace that aren't in my lane.

Reply:
@benjgil , FWIW, this was done after discussing with Ran and Gilad. (it was not the behavior previously). I don't remember the whole discussion, but in the use-cases we discussed it made sense.

Reply:
ok, i'll re-open that discussion elsewhere then as IMO it needs to be re-looked at